src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -41,6 +43,7 @@ public EventCard(Event event, int displayedIndex) {
         this.event = event;
         id.setText(displayedIndex + ". ");
         name.setText(event.getName().eventName);
+        description.setText(event.getDescription().toString());
```

Comment:
It doesn't really matter, but you can consider using `event.getDescription().description` as well.

Reply:
Sure